workplace-incoming-hook.go

```diff
@@ -32,6 +32,7 @@ var (
 	ThreadGitlab        string
 	ThreadTuleap        string
 	ThreadAppCenter     string
+	ThreadBackend       string
```

Comment:
it's not allowed here bro. Please use dynamic mapping

Reply:
Ok , I will change it